### PR TITLE
feat: Added conversion of Pydantic's 'description' field into DRF's '…

### PR DESCRIPTION
**Pull request by MrMika96**
…help_text' field
https://github.com/georgebv/drf-pydantic/issues/22

### CODE REVIEW

File: src/drf_pydantic/parse.py [hunk 1]
```diff
@@ -135,6 +135,10 @@ def _convert_field(field: pydantic.fields.FieldInfo) -> serializers.Field:
     if _default_value is not pydantic_core.PydanticUndefined:
         drf_field_kwargs["default"] = _default_value
 
+    # Adding description as help_text
+    if field.description:
```
Comment (georgebv) [inline src/drf_pydantic/parse.py:139]: I would instead check for two things explicitly: `None` or `pydantic_core.PydanticUndefined`. The truthy check is a bit ambiguous.

**Review by georgebv — CHANGES_REQUESTED** [review]
Thank you for the contribution. Can you please add unit tests to test fields with/without description?

**Comment by georgebv** [comment]
You can ignore the CI failure, codecov deprecated action version used to publish coverage.

File: src/drf_pydantic/parse.py (hunk 1 above)
Comment (MrMika96) [reply src/drf_pydantic/parse.py]: Resolved. Added both, so serializer field didnt look like this without description:
CharField(allow_null=True, default=None, help_text='None, required=False)

Comment (MrMika96) [reply src/drf_pydantic/parse.py]: I added test for fields into 'test_drf_field_kwargs'

**Comment by georgebv** [comment]
Everything looks good. Just format the code using ruff and we can merge.

**Comment by MrMika96** [comment]
> Everything looks good. Just format the code using ruff and we can merge.

Done.

**Review by georgebv — APPROVED** [review]

**Pull request merged**